lab2_task_1.py

```diff
@@ -0,0 +1,18 @@
+money_capital = 20000  # Подушка безопасности
+salary = 5000  # Ежемесячная зарплата
+spend = 6000  # Траты за первый месяц
+increase = 0.05  # Ежемесячный рост цен
+
+# TODO Посчитайте количество  месяцев, которое можно протянуть без долгов
+
+months = 0
+budget = money_capital + salary
+while budget - spend > 0:
+    if spend > budget:
+        break
+    budget -= spend
+    spend *= 1.05
+    budget += salary
+    months += 1
```

Comment:
Также можно рассмотреть альтернативный вариант решения: 
```suggestion
while money_capital + salary >= spend:
    money_capital = (money_capital + salary) - spend
    spend = spend * (1 + increase)
    months += 1
```
Тогда не надо будет вводить дополнительное условие проверки для остановки, при этом цикл сам остановится, когда верхнеуровневое условие не будет соблюдаться